packages/calendar/src/Calendar.ts

```diff
@@ -0,0 +1,449 @@
+import { ifDefined } from 'lit/directives/if-defined.js';
+import { when } from 'lit/directives/when.js';
+
+import { CalendarWeekday, daysInWeek } from './types.js';
+import styles from './calendar.css.js';
+
+import '@spectrum-web-components/action-button/sp-action-button.js';
+import '@spectrum-web-components/icons-workflow/icons/sp-icon-chevron-left.js';
+import '@spectrum-web-components/icons-workflow/icons/sp-icon-chevron-right.js';
+
+/**
+ * @element sp-calendar
+ *
+ * @slot prev-icon - The icon used in the _"Previous Month"_ button
+ * @slot next-icon - The icon used in the _"Next Month"_ button
+ *
+ * @event change - Announces when a day is selected by emitting a `Date` object
+ */
+export class Calendar extends SpectrumElement {
+    public static override get styles(): CSSResultArray {
+        return [styles];
+    }
+
+    @property({ type: Boolean, reflect: true })
+    padded = false;
+
+    @property({ type: Boolean, reflect: true })
+    disabled = false;
+
+    @property({ reflect: true, attribute: false })
+    selectedDate?: Date;
+
+    @property({ reflect: true, attribute: false })
+    min?: Date;
+
+    @property({ reflect: true, attribute: false })
+    max?: Date;
+
+    @state()
+    private _currentDate!: CalendarDate;
+
+    @state()
+    private _minDate!: CalendarDate;
+
+    @state()
+    private _maxDate!: CalendarDate;
+
+    private _languageResolver = new LanguageResolutionController(this);
+    private _locale!: string;
+    private _timeZone: string = getLocalTimeZone();
+
+    public get today(): CalendarDate {
+        return today(this._timeZone);
+    }
+
+    constructor() {
+        super();
+
+        this._setLocale();
+        this._setInitialCalendarDate();
+    }
+
+    protected override willUpdate(
+        changedProperties: PropertyValueMap<this>
+    ): void {
+        this._setLocale();
+
+        if (changedProperties.has('selectedDate')) {
+            this._setCurrentCalendarDate();
+        }
+
+        if (changedProperties.has('min')) {
+            this._setMinCalendarDate();
+        }
+
+        if (changedProperties.has('max')) {
+            this._setMaxCalendarDate();
+        }
+    }
+
+    protected override render(): TemplateResult {
+        return when(
+            this._currentDate,
+            () => html`
+                ${this.renderCalendarHeader()}${this.renderCalendarGrid()}
+            `
+        );
+    }
+
+    public renderCalendarHeader(): TemplateResult {
+        const monthAndYear = this._capitalFirstLetter(
+            this._formatDate(this._currentDate, {
+                month: 'long',
+                year: 'numeric',
+            })
+        );
+
+        return html`
+            <div class="spectrum-Calendar-header">
+                <!--
+                 * TODO: Attribute 'role="heading"' removed, due to error 'The "heading" role requires the attribute
+                 * "aria-level"'
+                -->
```

Comment:
Is this still current?

Reply:
Yes, all TODOs are listed in the README and where they occur